### ISSUE TITLE
TinyB documentation is misleading (BlueZ version)

### ISSUE THREAD

**Issue by vkolotov** [intel-iot-devkit/tinyb#58]
Please update your README.md file with a correct supported version of BlueZ. It mentions BlueZ 5.37, but tinyb 0.5.0 does not work with it. It fails on BluetoothGattCharacteristic.readValue(), something to do with DBus method could not be found. Bluez 5.43 works fine. I spent a fair amount of time to figure out what's wrong.

**Comment by gustavo84** [comment]
yes you have reason. I have the same problem and i have to look for, and you have update 5.42
